### DISCUSSION TITLE
Complete version of adwaitapod now available from Rockbox sources

### DISCUSSION THREAD

**Discussion by D0-0K** [D0-0K/adwaitapod#26] [Announcements]
Hi all!

 For the past year, adwaitapod with CJK fonts has only been available to download manually from the github repo. However with some recent changes I've made to the Rockbox themes site, the full version of adwaitapod can now be uploaded to Rockbox. 

 This means you can now install the theme, as well as updates, automatically using the rockbox utiltiy app. 

 This is a massive change, and really opens the door for more themes supporting more languages on Rockbox. I'd like to thank everyone who has been downloading the theme via github, providing feedback on fonts and so forth. The theme will continue to be available from here, alongside the accesibility and wallpaper packs, however I will be removing the CJK Font pack in several month.

Thanks again!
Dook

**Comment by teacherguy2020** [comment]
How do I get a theme to use Wallpaper.bmp? For example, awaitpod_dark_simplified? I see the Wallpaper.bmp file but it doesn’t seem to be used?

**Reply by teacherguy2020** [reply]
clarifying… I do see it used for example when I utilize USB. Can I use it on the While Playing display?

**Reply by D0-0K** [reply]
> How do I get a theme to use Wallpaper.bmp? For example, awaitpod_dark_simplified? I see the Wallpaper.bmp file but it doesn’t seem to be used?

So the Wallpaper.bmp file is a part of the Lockscreen feature. The lockscreen appears in 3 different conditions, the first being the USB screen as you discovered. The other two are when you switch the hold switch on when on the home menu or in the While Playing Screen, however the WPS display is usually turned off by default.

It can be switched on via the [user preference setting for WPS](https://github.com/D0-0K/adwaitapod/wiki/User-Preference-Settings#while-playing-screen-settings). However also take note of the next section for 'Lockscreen Style', as the wallpaper.bmp file is not used in the Always on Display version of the lockscreen.

**Reply by teacherguy2020** [reply]
Got you, thanks. Another question…is it possible to get the Quickscreen you talked about in your blog post into awaitapod? Can I do this myself? [https://d00k.net/blog/quickscreen_rockbox_tags/](url)

**Reply by teacherguy2020** [reply]
Nevermind, I found the post about editing the .cfg file, got it and it works great!
